<launch>

    <arg name="input_pcd"  default="/home/prostagma/orb_maps/small_map_raw.pcd" />
    <arg name="output_pcd" default="/home/prostagma/orb_maps/small_map_2.pcd"  />
    <arg name="filter"     default="ror" /> <!-- "sor", "ror", "none" -->

    <node name="pcd_filter" pkg="diplom" type="pcd_filter_node" output="screen">

        <param name="input_pcd_path"  type="string" value="$(arg input_pcd)" />
        <param name="output_pcd_path" type="string" value="$(arg output_pcd)" />

        <param name="filter_type"     type="string" value="$(arg filter)" />
        <param name="overwrite_output" type="bool" value="true" /> 

        <!-- Настройки SOR -->
        <param name="sor/mean_k"        type="int"    value="40" />
        <param name="sor/stddev_thresh" type="double" value="1.2" />

        <!-- Настройки ROR -->
        <param name="ror/radius_search"  type="double" value="0.5" />
        <param name="ror/min_neighbors"  type="int"    value="3" />

    </node>

</launch>